JSDemos/Demos/Tooltip/Overview/description.md

```diff
@@ -1 +1,16 @@
-The Tooltip component displays a&nbsp;tooltip for a&nbsp;specified element on&nbsp;the page. This demo shows how you can customize the tooltip and animate its appearance on&nbsp;the page.
\ No newline at end of file
+The Tooltip component displays a tooltip near an element on the page. To bind the Tooltip to the element, use the [target](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#target) property. 
+
+## Show and Hide the Tooltip
+
+Use two-way binding on the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip and assign a variable to this property. In the target element markup, assign a function that changes this variable to the specific [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) that trigger the visibility of a tooltip.
```

Comment:
> In the target element markup, assign a function that changes this variable to the specific [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) that trigger the visibility of a tooltip.

I cannot say I understand this sentence:
1. A function that changes this variable to the specific DOM events. Does the function replace the variable with specific DOM events?
2. DOM events that trigger the visibility of a tooltip. What do you mean by "trigger the visibility"?

Reply:
How about this (I rewrote the whole paragraph)?

> Specify [DOM events](https://developer.mozilla.org/en-US/docs/Web/Events#event_listing) in the target element markup that need to be triggered to change the Tooltip visibility. Bind the [visible](/Documentation/ApiReference/UI_Components/dxTooltip/Configuration/#visible) property of the Tooltip to a variable and use event handlers to toggle this variable.

Reply:
Now this sentence makes sense. Thank you.